examples/dial/example_dial.cpp

```diff
@@ -22,16 +22,15 @@
 #include <rpc/dial.hpp>
 
 using viam::robot::v1::Status;
-using Viam::SDK::Credentials;
-using Viam::SDK::Options;
+using namespace viam::cppsdk;
```

Comment:
This is fine in an example, but see below for how it should be done in SDK source files.